src/create-express-middleware.js

```diff
@@ -0,0 +1,32 @@
+import curry from 'ramda/src/curry';
+import { parse } from 'url';
+
+import { isActive } from './is-active';
+import { mergeFeatures } from './merge-features';
+import { getReqQueryFeatures } from './get-req-query-features';
+const setStatus = (res, isActive) =>
+  isActive ? res.status(200) : res.status(404);
+
+// ({ initialFeatures: [...String] }, { requiredFeature: String, ...methods }) => (req, res, next) => void
+export const createExpressMiddleware = curry(
+  ({ initialFeatures }, { requiredFeature, ...methods }) => (
+    req,
+    res,
+    next
+  ) => {
+    const parsedUrl = parse(req.url, true);
+    const { query } = parsedUrl;
+    const updatedFeatures = mergeFeatures(
+      initialFeatures,
+      getReqQueryFeatures({ query })
```

Comment:
Instead of using `getReqQueryFeatures` and wrapping it with an object you can use `getQueryFeatures` and pass it the query

Reply:
Not sure how I missed that. Fixed now.